### ISSUE TITLE
Make DMs easier to differentiate from normal feed posts in dark mode

### ISSUE THREAD

**Issue by sarajw** [elk-zone/elk#859]
Maybe a brighter outline - or if it's all done with opacities (sorry not checked the code), maybe a more opaque value on the background. 

Anything to help it show up more - to see the difference currently I have to turn my phone's brightness way up.

**Comment by TheWebTech** [comment]
How about corner plus an envelope icon? It would require alt/title text on the icon so when you hovered it you'd get an obvious "This is a DM" message.

**Comment by TheWebTech** [comment]
Want to verify this wasn't fixed already.

The current state looks like this:
![DM sent within a public thread](https://user-images.githubusercontent.com/5492388/212483737-62a48ea4-3b5d-4c44-8902-d315526c62ad.png)
![DM as received by user within public thread](https://user-images.githubusercontent.com/5492388/212483773-829b6be8-6c55-4956-ac33-8a1d26e47b1c.png)

**Comment by sarajw** [comment]
Yes this has already changed since I wrote this issue, and it's much better than it was.
